### PR TITLE
feature: support set home dir in dfget config file.

### PR DESCRIPTION
**Pull request by zcc35357949**
Signed-off-by: zhouchencheng <zhouchencheng@bilibili.com>

### Ⅰ. Describe what this PR did
Dfget server uses current user's home directory as root workdir, and it is not configurable now.
I wish that I can change the dfget work dir by dfget's config file.

### Ⅱ. Does this pull request fix one issue?
None

### CODE REVIEW

File: cmd/dfget/app/root.go [hunk 1]
```diff
@@ -64,16 +64,16 @@ func init() {
 
 // runDfget does some init operations and starts to download.
 func runDfget() error {
+	// get config from property files
+	initProperties()
+
```
Comment (yunfeiyanggzq) [inline cmd/dfget/app/root.go]: you make init config before init log,I have see when init config  it  put out log by `Logrus.debugf`,so there may be something wrong.There are no place the log  put out. You'd  better make sure it

File: cmd/dfget/app/root.go [hunk 2]
```diff
@@ -135,6 +155,24 @@ func initProperties() {
 	if cfg.ClientQueueSize == 0 {
 		cfg.ClientQueueSize = properties.ClientQueueSize
 	}
+
+	currentUser, err := user.Current()
+	if err != nil {
+		printer.Println(fmt.Sprintf("get user error: %s", err))
+		os.Exit(config.CodeGetUserError)
+	}
+	cfg.User = currentUser.Username
```
Comment (Starnop) [inline cmd/dfget/app/root.go]: In fact, `cfg.User` is not a necassary field. I will open a new PR to fix that. Make a record here.

File: cmd/dfget/app/root.go [hunk 3]
```diff
@@ -64,15 +71,24 @@ func init() {
...
+	propResults := initProperties()
+
 	// initialize logger
 	if err := initClientLog(); err != nil {
 		return err
 	}
 
-	cfg.Filter = transFilter(filter)
+	// print property load result
+	for _, propRst := range propResults {
```
Comment (Starnop) [inline cmd/dfget/app/root.go]: It's weird. However, I have no idea to make it elegant at present. 😢  @zcc35357949 Could you please add a TODO record here in the source code? THX.

**Comment by Starnop** [comment]
@zcc35357949 Could you please update your code and make it pass the CI? THX.

**Comment by Starnop** [comment]
LGTM.

**Pull request merged**